wallet/walletrpc.c

```diff
@@ -288,9 +289,11 @@ static struct command_result *json_prepare_tx(struct command *cmd,
 		 *   native segwit, nlocktime set to tip, and sequence set to
 		 *   0xFFFFFFFE by default. Other wallets are likely to implement
 		 *   this too).
-		 * FIXME: Do we want to also fuzz this like bitcoind does ?
 		 */
 		locktime = cmd->ld->topology->tip->height;
+		/* Eventually fuzz it too. */
+		if (pseudorand(10) == 0)
+			locktime -= (u32)pseudorand(100);
```

Comment:
No range check for negative block heights?  Though that may be overly paranoid at this point for Bitcoin, imagine a new blockchain (bleah!) that supports Lightning from genesis...  Naaaa keep this.